images/linux/scripts/tests/Toolset.Tests.ps1

```diff
@@ -0,0 +1,69 @@
+Describe "Toolset" {
+    $tools = (Get-ToolsetContent).toolcache
```

Comment:
so in fact we are testing toolcache here. Maybe makes sense to rename file and test suite to Toolcache?

Reply:
Thank you for your point, @AlenaSviridenko. I'd prefer to leave current naming as it is for now. Just to be consistent with other OS's.
cc: @maxim-lobanov